packages/iov-bns/src/context.ts

```diff
@@ -72,8 +72,10 @@ export class Context {
         sender: encodeBnsAddress(addressPrefix(this.chainData.chainId), ensure(swap.sender)),
         recipient: encodeBnsAddress(addressPrefix(this.chainData.chainId), ensure(swap.recipient)),
         hash: hash,
-        amounts: ensure(swap.amount).map(coin => decodeAmount(coin)),
-        timeout: { height: asNumber(swap.timeout) },
+        // amounts: ensure(swap.amount).map(coin => decodeAmount(coin)),
+        // TODO: read this is a second query
+        amounts: [],
```

Comment:
Why was amount removed from Escrow? This seems to be very bad for the client

Reply:
Because it was not accurate in the general case. Let's discuss this more off-line.

Multisigs, distribution, etc will need two queries, as we do not have triggers in weave to call custom functionality when a contract receives tokens (as in ethereum). We can debate if we need such.

Out of scope of this PR, but let's have some design discussions on this topic